website/app/styles/pages/application/header.scss

```diff
@@ -43,10 +43,6 @@
   padding: 0 24px;
   color: var(--doc-color-white);
   background: var(--doc-color-black);
-
-  @include breakpoint.with-fixed-sidebar() {
-    gap: 32px;
```

Comment:
The gap between the header and the edge of the page is always 24px now.

Reply:
This reduced the space between the logo and the nav links to being 16px always, which I think we don't want to do (it's too tight right now). 
<img width="1478" alt="image" src="https://github.com/user-attachments/assets/62297fff-309d-4890-9b62-ef9c204c004c">

I think removing this is removing essentially this?
<img width="253" alt="image" src="https://github.com/user-attachments/assets/ce968069-51d9-4362-a621-892782bc4f3e">